scheduler/ipp.c

```diff
@@ -9374,6 +9374,9 @@ restart_job(cupsd_client_t  *con,	/* I - Client connection */
 		"Restarted by \"%s\" with job-hold-until=%s.",
                 username, attr->values[0].string.text);
     cupsdSetJobHoldUntil(job, attr->values[0].string.text, 0);
```

Comment:
Yes, the last argument can be changed to "1" here, good catch!

Reply:
Done.